### PR TITLE
qemu.tests.block_resize: Add rescan step after shrink disk for raw forma...

### PR DESCRIPTION
**Pull request by spcui**
...t

1. Add rescan step after shrink disk for raw format
2. Update some related parameters

### CODE REVIEW

**Comment by spcui** [comment]
ID: 1205029

File: qemu/tests/block_resize.py [hunk 1]
```diff
@@ -111,9 +113,16 @@ def compare_block_size(session, block_cmd, block_pattern):
...
-        tmp = vm.monitor.block_resize(data_image_dev, block_size)
+
+        vm.monitor.block_resize(data_image_dev, block_size)
+
         if need_reboot:
             session = vm.reboot(session=session)
+
+        # We need to rescan disk in diskpart after block_resize in windows
+        if disk_rescan_cmd:
+            session.cmd(disk_rescan_cmd)
```
Comment (ypu) [inline qemu/tests/block_resize.py]: Hi seems we don't need rescan disk it we already reboot the guest. So maybe we can make it an elif branch here.

**Comment by spcui** [comment]
@ypu thanks for your comment, already updated now, could you help review again?

**Comment by ypu** [comment]
LGTM
Acked-by: Yiqiao Pu <ypu@redhat.com>

**Comment by ypu** [comment]
Thanks @spcui will merge it.

**Pull request merged**